sap/rfc/user.py

```diff
@@ -176,6 +176,11 @@ def _rfc_params_add_password(self, params):
         add_to_dict_if_not_present(password, 'BAPIPWD', '')
         params['PASSWORD'] = password
 
+    def _rfc_params_add_alias(self, params):
```

Comment:
Please, use the new method _rfc_params_add_alias also in the method build_rfc_params.